web/Makefile

```diff
@@ -20,7 +20,7 @@ SUFFIXES:
 blob/files.go: $(shell find templates/ static/ -type f)
 	# Note that embed-static.sh excludes map files and the
 	# non-minified bootstrap files.
-	../scripts/embed-static.sh static templates | $(GOFMT) > $@
+	go generate ./blob
 
 clean:
```

Comment:
I agree with @fabxc that `files.go` should be checked in now, so how about just getting rid of the `clean` target here and also removing the reference to it in https://github.com/prometheus/prometheus/blob/master/Makefile#L58.